### DISCUSSION TITLE
CSS accent-color

### DISCUSSION THREAD

**Discussion by chrfickinger** [tailwindlabs/tailwindcss#5846] [Ideas]
Hey guys,
possibly useful for future version of tailwindcss. 

Check the following link for more details:
https://web.dev/accent-color/

Cheers
Chris

**Comment by lukewarlow** [comment]
This has already been added and will be released as part of Tailwind 3. Checkout the alpha releases to give it a try. Alternatively if you want to use them in a Tailwind 2 project, my [tailwind-accent-color](https://github.com/lukewarlow/tailwind-accent-color) plugin can be used .
